docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -136,12 +136,22 @@ You're all set to use the custom version of the Expo client, containing features
 
 # Troubleshooting
 
-## Push Notifications aren't working
+## Fixing Disabled Services
+
+### Push Notifications
+
+**Note:** Push Notifications are temporarily disabled until we support API tokens for our Push Notification system.
```

Comment:
```suggestion
**Note:** Push Notifications are currently unavailable with ad hoc clients until we complete our work to add an extra authentication layer to the Expo Push Notification service.
```

And comment out the rest of this push notifications guide since it's just confusing to have it.